packages/speech/src/helpers.js

```diff
@@ -0,0 +1,113 @@
+/*!
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*!
+ * @module speech/helpers
+ */
+
+'use strict';
+
+var pumpify = require('pumpify');
+var streamEvents = require('stream-events');
+var through = require('through2');
+
+/*!
+ * Return a dictionary-like object with helpers to augment the Speech
+ * GAPIC.
+ *
+ * @return {Object} - An object with keys and functions which are placed
+ *   onto the pure GAPIC.
+ */
+module.exports = () => {
+  var methods = {};
```

Comment:
@lukesneeringer These Speech helpers are missing the file loading helpers that the Vision partial gapic helpers have: https://github.com/lukesneeringer/google-cloud-node/blob/ada694f1ced83e8d46d685ed8d6a7d20f204e494/packages/vision/src/helpers.js#L51